docs/src/inflation.md

```diff
@@ -0,0 +1,45 @@
+# Inflation 
+Inflation is a method which systematically adds stochastic perturbations to the parameter ensemble in the form of gaussian noise. 
+Following the standard Kalman update, the parameter ensemble is perturbed in the following manner:
+```math
+    u_{n+1} = u_n + \zeta_{n} \qquad (1)
```

Comment:
This is just the additive inflation. Perhaps just write down that the mulitplicative inflation based on expanding the ensemble away from it's mean in a deterministic way, while additive inflation is based on adding this noise term?